src/insight/metrics/metrics.py

```diff
@@ -334,7 +352,7 @@ class EarthMoversDistanceBinned(TwoColumnMetric):
     an ordinal range. If the latter, they must have equal binning.
 
     Args:
-        bin_edges: Optional; If given, this must be an iterable of bin edges for x and y,
+        bin_edges: ty.Optional; If given, this must be an iterable of bin edges for x and y,
```

Comment:
nit: no need to indicate that `Optional` has been imported from the `typing` module in a docstring. I'll leave this to you though as it's a matter of style and potentially consistent across this repo

Reply:
Right now it's consistent across all docstrings -- i'll leave it as is, but we can remove `ty.` from docstrings if you want to

Reply:
I'm not fussed, so if it's consistent and that's the styling you're going for then that's fine!